config/peerpods/podvm/bootc/podvm-git-buildconfig.yaml

```diff
@@ -0,0 +1,32 @@
+# This will build according to git source and push to the local cluster's registry:
+# image-registry.openshift-image-registry.svc:5000/openshift-sandboxed-containers-operator/podvm-bootc
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: podvm-bootc
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  lookupPolicy:
+    local: false
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: bootc-image-build
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: podvm-bootc:latest
+  source:
+    contextDir: "config/peerpods/podvm/bootc"
+    git:
+      uri: 'https://github.com/openshift/sandboxed-containers-operator.git'
+      ref: devel
```

Comment:
Reading again, I'm concerned of something else... if this is always pointing to devel, that means that anyone in production using podvm images from bootc would be using code from devel branch? If so, maybe we can add a `# TODO-RELEASE: ` comment on top of it, so we can easly grep and replace values before creating the release branch?

Reply:
would it make sense to switch ref to release-1.9? (not exist yet but still..)

Reply:
I think that as long the code exists on devel branch, it should be devel. But we need a strategy to when we are creating the "release branch". That is the reason I mentioned the # TODO-RELEASE. But I think @gkurz and @c3d had used `## OSC_VERSION` before. not sure how that tag is used and if have the same goal, tho.